### PR TITLE
feat/complete signup

### PR DESCRIPTION
**Pull request by new5558**
To test:
- Try sign in in more than 2 pages
- Try update additional info
- Can play interest but not finished yet

### CODE REVIEW

**Review by poom-kitti — DISMISSED** [review]
Go review dev-navbar & add it

**Review by Seqlet — APPROVED** [review]
LGTM

File: src/commons/components/page-navbar/PageNavbar.vue [hunk 1]
```diff
@@ -43,7 +43,10 @@
       >Login / Signup</base-button
     >
     <section v-else class="flex items-center">
-      <div class="mr-3 h-5 w-5 border-2 border-primary rounded-full">
+      <div
+        v-if="imgUrl"
+        class="mr-3 h-5 w-5 border-2 border-primary rounded-full"
+      >
```
Comment (poom-kitti) [inline src/commons/components/page-navbar/PageNavbar.vue]: If there is no img, there should still be the circle but with initial in circle
image

File: src/commons/utils/validForm.ts [hunk 2]
```diff
@@ -47,5 +47,5 @@ export function validateProvince(province: string) {
  * Check whether address is not empty
  */
 export function validateAddress(address: string) {
-  return address.trim() !== "";
+  return true;
```
Comment (poom-kitti) [inline src/commons/utils/validForm.ts]: Why return only true?

File: src/commons/components/page-navbar/usePageNavbar.ts [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
+  const { isSignIn: isLogIn, logout, user } = useUser();
+  const imgUrl = computed(() => {
+    return user.value.profilePictureUrl;
+  });
+
+  const nameShown = computed(() => {
+    if (isEmpty(user.value)) {
+      return "";
+    }
+    return user.value.firstName + user.value.lastName?.charAt(0) + ".";
```
Comment (poom-kitti) [inline src/commons/components/page-navbar/usePageNavbar.ts]: Add space between first name and last name

**Review by poom-kitti — APPROVED** [review]
LGTM

**Pull request merged**